docs/general/polkadot-vault.md

```diff
@@ -0,0 +1,126 @@
+
+|                                                                                                             Ledger                                                                                                             |                                                                                                                                                                            Polkadot Vault                                                                                                                                                                             |
+| :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|              Hardware designed to stay offline and be secure. Certified by French cybersecurity agency [ANSSI](https://www.ssi.gouv.fr/en/cybersecurity-in-france/the-national-cybersecurity-agency-of-france/).               |                                                                                                                 Hardware not designed to stay offline. The user must switch off all inbound and outbound connections (network, wifi, Bluetooth, NFC).                                                                                                                 |
+|                                 Hardware is resistant to side-channel attacks via [Secure Element](https://www.ledger.com/academy/security/the-secure-element-whistanding-security-attacks/).                                  |                                                                                                                                                If there is no wired connection, side-channel attacks are not possible.                                                                                                                                                |
+|                                                                              One mnemonic seed phrase for all intra- and inter-currency accounts.                                                                              | One mnemonic for each account or one mnemonic for multiple accounts via [account derivation](../learn/learn-account-advanced.md#derivation-paths) or default [Substrate address format](../learn/learn-account-advanced.md#address-format) (not recommended to use the same account on different chains). For `n` chains, a minimum of `n` mnemonics are recommended. |
```

Comment:
> (not recommended to use the same account on different chains). For `n` chains, a minimum of `n` mnemonics are recommended.

One could argue that this is one of the great features of Substrate and it _is_ recommended. But really there's no recommendation here; it depends on the user's needs. Check [this article](https://support.polkadot.network/support/solutions/articles/65000103707-can-i-use-the-same-account-on-polkadot-kusama-and-parachains-#Should-I-use-the-same-account-across-different-chains?).